osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -32,3 +32,16 @@ def get_list_or_none(klass, *args, **kwargs):
     if not object_list:
         return None
     return object_list
+
+
+def get_actual(obj):
+    """Expands `obj` to the actual object type.
+    """
+    for name in dir(obj):
+        try:
+            attr = getattr(obj, name)
+            if isinstance(attr, obj.__class__):
```

Comment:
Can https://django-model-utils.readthedocs.org/en/latest/managers.html#inheritancemanager be of help, here?